libs/api/utils/src/lib/api-utils.ts

```diff
@@ -1,8 +1,31 @@
+import { prisma } from 'apps/coursition/prisma/prismaClient'
+import { parseApiKey } from '../core/api/parseApiKey'
+import { isDateBeforeNow } from '../core/date/isDateBeforeNow'
```

Comment:
**suggestion:** After date is moved to `utils`, we should use import alias, e.g. `from '@nmit-coursition/utils'`. It shouldn't be allowed to import cross-package without aliases.